dotCMS/src/main/java/com/dotmarketing/business/PermissionAPI.java

```diff
@@ -786,16 +785,20 @@ void assignPermissions(List<Permission> permission, Permissionable permissionabl
 	 */
 	boolean doesUserHavePermissions(PermissionableType permType, int permissionType, User user) throws DotDataException;
 
-	/**
-	 *
-	 * @param parent
-	 * @param permissionable
-	 * @param user
-	 * @param respectFrontendRoles
-	 * @throws DotDataException
-	 * @throws DotSecurityException
-	 */
-	void permissionIndividually(Permissionable parent, Permissionable permissionable, User user, boolean respectFrontendRoles) throws DotDataException, DotSecurityException;
+    /**
+     * This method should be called when we want to break permission inheritance. It will find
+     * parent permissions and apply all those permissions to the permissionable.
+     */
+    void permissionIndividually(Permissionable parent, Permissionable permissionable, User user,

```

Comment:
I think we should create two a second method call permissionIndividuallyRespectFrontEndRoles; it could be a way to avoid flag parameter. The original will be in false.